config/crd/bases/dspipelines.opendatahub.io_dspipelines.yaml

```diff
@@ -0,0 +1,163 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.10.0
+  creationTimestamp: null
+  name: dspipelines.dspipelines.opendatahub.io
```

Comment:
Does this name make sense? Should it be `dspipelines.dspipelines.opendatahub.io`?

Reply:
the resource is <dspipelines> the api group is <dspipelines.opendatahub.io>, so depends on what we want, this is a separate jira task as well, so it's covered elsewhere